api/grpc/v1/agent.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package flightctl.v1;
```

Comment:
Can this be more mature than our user-facing API which is v1alpha1?

Reply:
Yes, in this case it's the format of this API.

When requesting objects related to the user-facing API, that would be ser/deserialized in json, and we would explicitly provide an api_version parameter from the agent.